grid2op/multi_agent/subGridObjects.py

```diff
@@ -301,6 +309,101 @@ def init_grid_from_dict_for_pickle(name_res, orig_cls, cls_attr):
     def _get_full_cls_str(cls):
         return GridObjects._get_full_cls_str(cls)
     
+    @classmethod
+    def _check_sub_id(cls):
```

Comment:
what the hell is that ?

No no no you should not have anything like this here.

I don"t know what this method does, what it is supposed to do etc.

Please re use api only from grid2op without having to recode / copy paste / whatever... I don't know why you overload that.

I told you to do "like i did for generators and loads" and I really don't see why you have to do that while i did not...

Reply:
Ok done